tests/main/writable-areas/task.yaml

```diff
@@ -1,5 +1,9 @@
 summary: Check that snap apps and services can write to writable areas.
 
+environment:
+    SNAP_REEXEC/norexec: 0
+    SNAP_REEXEC/withreexec: 1
+
```

Comment:
All tests are designed to run with re-exec. We put snap/snapd into the core snap so that all tests use them. There is also e.g. "install-sideload" that will run with both re-exec on and off and that installs and runs  hello-world (test-snapd-tools to be exact).